inputs/org.osbuild.containers

```diff
@@ -220,9 +231,24 @@ class ContainersInput(inputs.InputService):
         source = store.source("org.osbuild.containers")
         images = {}
 
+        storage_conf = containers.get_host_storage()
```

Comment:
I wonder if we should be more defensive here and read the host storage only if `data.get("input-type") == "org.osbuild.containers-storage"`. I'm a bit worried that `containers.get_host_storage()` might fail for unrelated reasons and that would result into failed builds even for manifests that don't need to take any inputs from the container storage.

Reply:
Oh actually you're 100% right. It will fail because the TOML package isn't present. I just didn't want to be reading the host config inside the loop. I'll fix this